cmd/root.go

```diff
@@ -2,20 +2,18 @@ package cmd
 
 import (
 	"fmt"
-	"github.com/go-flutter-desktop/hover/internal/log"
 	"os"
 	"os/signal"
 
-	"github.com/logrusorgru/aurora"
+	"github.com/go-flutter-desktop/hover/internal/log"
+
 	"github.com/spf13/cobra"
 )
 
-var au aurora.Aurora
-var colors bool
```

Comment:
`colors` doesn't need to be global

Reply:
False, colors needs to be global. Had a hard time figuring this out because we have 2 init functions that runs `cobra.OnInitialize`.

Reply:
I've removed the `cobra.OnInitialize` inside `cmd/common.go` and added a `initHover` function.
It's cleaner and it works!